pom.xml

```diff
@@ -56,8 +59,32 @@
     <jbpm.wb.repo.url>${git.server.url}/jbpm-wb</jbpm.wb.repo.url>
     <kie.wb.common.repo.url>${git.server.url}/kie-wb-common</kie.wb.common.repo.url>
     <kogito.examples.repo.url>${git.server.url}/kogito-examples</kogito.examples.repo.url>
+
+    <!-- properties affecting maven modules resolution using resolve-includes.groovy script -->
+    <maven.modules.resolution.timeout>600000</maven.modules.resolution.timeout>
```

Comment:
is this miliseconds?

Reply:
yes, will rename the prop to indicate that.